scripts/ci/gitlab/pipeline/build.yml

```diff
@@ -90,9 +90,6 @@ build-linux-substrate:
   variables:
     # this variable gets overriden by "rusty-cachier environment inject", use the value as default
     CARGO_TARGET_DIR:              "$CI_PROJECT_DIR/target"
-  needs:
```

Comment:
Not needed because the pipeline will be stopped by `pipeline-stopper` in case any check fails